internal/sdk/cloudian/sdk.go

```diff
@@ -200,6 +200,16 @@ func (client Client) CreateUser(ctx context.Context, user User) error {
 
 	switch resp.StatusCode() {
 	case 200:
+		// Delete the (one) access key boodstrapped with new user
+		creds, err := client.ListUserCredentials(ctx, user)
+		if err != nil {
+			return err
+		}
+		for _, cred := range creds {
+			if err := client.DeleteUserCredentials(ctx, cred.AccessKey); err != nil {
+				return err
+			}
+		}
```

Comment:
Why do we delete something on create?

Reply:
When newly created, users have an access key (not provisioned by an AccessKey)

Reply:
If left there, the limit of usable access keys is reduced from 5 to 4, and we cannot require all access keys to be deleted before deleting the user.

Reply:
Ok, thanks for the explanation. Is there a way to disable the auto-creation of this (non-managed) access key in Cloudian? If not, I would suggest a full rewrite where an access key is not managed separately, but instead "part of user". 😉 I think this could REALLY messy.

Reply:
Agree it's messy, but don't think it's possible to disable :/